core/templates/dev/head/domain/utilities/language-util.service.ts

```diff
@@ -137,7 +137,8 @@ export class LanguageUtilService {
     return this.getAutogeneratedAudioLanguages('autogen-lang-code')
       .hasOwnProperty(audioLanguageCode);
   }
-  getAutogeneratedAudioLanguage(explorationLanguageCode: string): string {
+  // TODO(#7165): Replace any with correct type.
+  getAutogeneratedAudioLanguage(explorationLanguageCode: string): any {
```

Comment:
What's going on here?

Reply:
This function is being used in the audioTranslationService.
```
this.languageUtilService.getAutogeneratedAudioLanguage(this._explorationLanguageCode).id;
``` 
Hence it doesn't returns a string, but an object having some properties. Is that clear? @bansalnitish